masterlist.yaml

```
@@ -14987,6 +15001,30 @@ plugins:
         util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
 
+  - name: 'HandtoHand.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/59790/' ]
```

A regex plugin can be used to distribute shared metadata (e.g., locations) to multiple plugins (n.b., plugin names are case-insensitive):
```suggestion
  - name: '(HandtoHand(Armor|Locksmith)?|H2HBooksAndTrainers)\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/59790/'
        name: 'Hand to Hand - An Adamant Addon'
  - name: 'HandtoHand.esp'
```